assets/csidriveroperators/aws-ebs/01_namespace.yaml

```diff
@@ -1,7 +0,0 @@
-apiVersion: v1
-kind: Namespace
-metadata:
-  name: openshift-cluster-csi-drivers
-  annotations:
-    include.release.openshift.io/self-managed-high-availability: "true"
-    openshift.io/node-selector: ""
```

Comment:
Next time we are here we should rename the other files (the first asset file is prefixed with `02_`.